style.css

```diff
@@ -80,3 +83,172 @@ nav #myLogo {
   font-size: 18px;
   color: #6070ff;
 }
+
+.grid-box {
+  display: flex;
+  margin-left: auto;
+  margin-right: auto;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: #e5e5e5;
+  border-radius: 8px;
+  width: 100%;
+  gap: 88px;
+  padding: 104px 0;
+}
+
+.icon {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 20px;
+  padding: 0;
+}
+
+.box {
+  background-color: #fff;
+  gap: 10px;
+  border-radius: 15px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 16px;
+  width: min(327px, 80vw);
+  border: 2px solid #dfe1e6;
+}
+
+.banner {
+  border-radius: 8px;
+  width: min(327px, 80vw);
+}
+
+input {
+  color: #396df2;
+  font-size: 17px;
+  font-weight: 500;
+  background-color: #fff;
+  border-color: #6070ff;
+  border-radius: 8px;
+  padding: 8px 5px;
+  display: flex;
+  width: min(127px, 80vw);
+}
```

Comment:
- [ ] It looks like you are not aware of the various styles recommended for the button states and interaction like `hover`, `active` etc. Here is the pallet from Figma:
![image](https://user-images.githubusercontent.com/99715640/212442153-818d9c3d-d83a-42e9-bf9e-95393e30b3ae.png)
- [ ] Please inspect it and implement it to have a sparkling design with amazing user experience 😉  It sounds cool, right?